assets/aws/files/bin/teleport-renew-cert

```diff
@@ -14,6 +14,27 @@ if [ ! -f /etc/teleport.d/role.auth ] && [ ! -f /etc/teleport.d/role.all ]; then
     exit 0
 fi
 
+# Fetching certbot state
+aws s3 sync --exact-timestamps s3://${TELEPORT_S3_BUCKET} /etc/letsencrypt/ --sse=AES256
+
+# s3 does not support symlinks, we have to create them after the sync, else certbot will fail.
+# live/ symlinks point to the latest archive/<domain>/<object>XX.pem where XX is incremented at each cert-renewal.
+# The last iteration is retrieved by listing all fullchains, sorting them by iteration (this is not alphabetical order
+# because fullchain10.pem should be greater than fullchain2.pem). We finally strip the id from the filename.
+ARCHIVE_NUMBER="$(
+  find /etc/letsencrypt/archive/hugo-tf.teleportdemo.net/ -iname "fullchain*.pem" \
```

Comment:
This hostname shouldn't be hardcoded here

Reply:
Oh ****, bad copy-paste thanks